### ISSUE TITLE
Add gradient clipping

### ISSUE THREAD

**Issue by shairoz-deci** [Deci-AI/super-gradients#146]
### Is your feature request related to a problem? Please describe.
Some training recipes require gradient clipping especially when transfer learning from pretrained models. Such option should be added to training params

### Describe the solution you'd like
add clip_grad_norm to training params
